test/tests/api/v2_0/nodes_tests.py

```diff
@@ -401,3 +404,27 @@ def test_node_tags_del(self):
             assert_equal(200, c.status, message=c.reason)
         assert_raises(rest.ApiException, Api().nodes_del_tag_by_id, 'fooey',tag_name=['tag'])
 
+
+    @test(groups=['nodes_tag_masterDelete'], depends_on_groups=['node_tags_delete'])
+    def test_node_tags_masterDel(self):
+        """ Testing DELETE:api/2.0/nodes/tags/:tagName """
+        codes = []
+        Api().nodes_get_all()
+        rsp = self.__client.last_response
+        nodes = loads(rsp.data)
+        codes.append(rsp)
+        self.test_node_tags_patch()
+        t = 'tag3'
+        for n in nodes:
+            Api().nodes_master_del_tag_by_id(tag_name=t)
```

Comment:
Is this only verifying that master delete doesn't delete invalid tags?  I don't see where it attempts to delete the tags set by test_node_tags_patch().